site/content/docs/user/quick-start.md

```diff
@@ -366,8 +366,7 @@ kubeadmConfigPatches:
 {{< /codeFromInline >}}
 
 #### IPv6 clusters
-You can run ipv6 only clusters using `kind`, but first you need to enable ipv6 in your docker daemon by editing `/etc/docker/daemon.json` [as described here][docker enable ipv6].
```

Comment:
don't edit docs based on an unreleased PR. file a second PR and we'll merge when the release is done.

Reply:
yeah, this was waiting for https://github.com/kubernetes-sigs/kind/pull/1526